vaadin-grid-doc.html

```diff
@@ -423,6 +423,7 @@
 
       /**
        * Contents of the cell.
+       * Note: Assigning an Element object as content disables manual sorting for that cell.
```

Comment:
I would rephrase that to “Assigning an HTML element reference as the content for the default header row cell disables end users sorting for that column.” to be absolutely precise what it does.